config/vufind/combined.ini

```diff
@@ -33,6 +33,15 @@
 ;         the side column. If set to false, side recommendations will be suppressed
 ;         (default). Adding the same recommendation module to multiple sections is
 ;         unsupported.
+; include_recommendations_noresults = If set to true, standard 'top'
```

Comment:
This should say 'norecommend' instead of 'top', shouldn't it?

Reply:
D'oh.  Fixed.